docs/parity-status.md

```diff
@@ -1,77 +1,32 @@
 id: parity-status
-title: Component and Module API Parity Status
+title: API Parity
 ---
 
-This doc reports the status progress of core components modules/APIs integration for vnext. The high priority components and modules tracked here will be in alignment with the [React Native Lean Core](https://github.com/facebook/react-native/issues/23313) effort, others not in Lean Core we plan on shipping as separate packages in [future milestones](https://github.com/microsoft/react-native-windows/milestones).
+[React Native Components and APIs](https://reactnative.dev/docs/components-and-apis) that are a part of the [React Native Lean Core](https://github.com/facebook/react-native/issues/23313) effort are now all supported in React Native for Windows.
 
-If there is a discrepancy here, or features are not being tracked that should be, please [submit an issue under vnext](https://github.com/microsoft/react-native-windows/issues/new?labels=vnext&template=vnext.md) to let us know.
+For a more closely monitored look at our work in progress, check out the [API Completion](https://github.com/microsoft/react-native-windows/labels/API%20Completion) label on issues in our repo.
 
-You can keep track of our component and module API parity more closely on our [GitHub project borad](https://github.com/microsoft/react-native-windows/projects/7).
+> Please feel free to leave comments or give us a 👍 on issues to help us prioritize better!
 
-### Components
-Below are a list of components and their status towards parity between the lean core components you'd find on iOS and Android and the functionality we have today.
+A few methods or props may be missing on some types and we are actively working to add support for those in our [upcoming milestones](https://github.com/microsoft/react-native-windows/milestones).
 
-|Component| `vnext` version Status | `vnext` Issues remaining | `current` version Status |
-|:-|:-|:-|:-|
-|ActivityIndicator|**Complete**|-|**Complete**|
-|Button|**Complete**|-|**Complete**|
-|FlatList|**Complete**|-|**Complete**|
-|Image|Partial|[Image project board](https://github.com/microsoft/react-native-windows/projects/18)|Partial|
-|ScrollView|Partial|[ScrollView project board](https://github.com/microsoft/react-native-windows/projects/17)|Partial|
-|Switch|**Complete**|-|**Complete**|
-|Text|**Complete** *(Some properties not supported due to device platform differences, docs coming soon)*|**Complete**|**Complete**|
-|TextInput|Partial|[TextInput project board](https://github.com/microsoft/react-native-windows/projects/20)|Partial|
-|KeyboardAvoidingView|**Complete**|-|Not supported|
-|View|Partial (some Style props left)|[View Style props project board](https://github.com/microsoft/react-native-windows/projects/19)|**Complete**|
-|VirtualizedList|Partial|No Issues Logged|**Complete**|
+If you an unsupported API that should be tracked, please [submit an issue](https://github.com/microsoft/react-native-windows/issues/new/choose) to let us know.

```

Comment:
"If you **encounter** an unsupported"...

Reply:
oops!